packages/core-react/src/components/Button/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+export default function Button({
+  Tag = 'button',
+  children,
+  type,
+  isCompact,
+  isDanger,
+  disabled,
```

Comment:
should we standardize on `boolean` props? either `isSomething` or just `something`. For example, we have `disabled` and `isCompact` here. I personally prefer `compact`, `danger`, and `disabled`

Reply:
reason i left out disabled is because it's native api.. but yea you're approach makes sense. the prefixes might be overkill

Reply:
I agree with ramin here :)

Reply:
cool, thanks for the input